fit/core-reference/pom.xml

```diff
@@ -1212,15 +1212,15 @@ under the License.
       <id>oracle-it</id>
       
       <properties>
-        <jdbcdriver.groupId>com.oracle.jdbc</jdbcdriver.groupId>
-        <jdbcdriver.artifactId>ojdbc8</jdbcdriver.artifactId>
+        <jdbcdriver.groupId>com.oracle.ojdbc</jdbcdriver.groupId>
+        <jdbcdriver.artifactId>ojdbc10</jdbcdriver.artifactId>
       </properties>
 
       <dependencies>
         <dependency>
-          <groupId>com.oracle.jdbc</groupId>
-          <artifactId>ojdbc8</artifactId>
-          <version>12.2.0.1</version>
+          <groupId>${jdbcdriver.groupId}</groupId>
```

Comment:
I know this is also correct, but would you mind reverting back to explicit groupId / artifactId for Oracle JDBC driver?

The two properties are mainly used to declare `cargo-maven-plugin`'s dependencies.